src/locale/locales/de/messages.po

```diff
@@ -100,11 +100,11 @@ msgstr "Konto stummgeschaltet"
 #: src/components/moderation/ModerationDetailsDialog.tsx:93
 #: src/lib/moderation/useModerationCauseDescription.ts:91
 msgid "Account Muted"
-msgstr "Konto Stummgeschaltet"
+msgstr "Konto stummgeschaltet"
```

Comment:
Our initial thought here was that the English translation is uppercased, hence we did the same for the German translation. I guess that's open for debate? I'm fine with both versions.

Reply:
I understood it as some sort of title for a dialog/modal, but titles aren't all uppercased in German like they often are in English (see e.g. Wikipedia article structures)

Reply:
Sure, I'm fine with your approach